### ISSUE TITLE
How to get jwks.json for Cognito on Docker

### ISSUE THREAD

**Issue by nakata72** [spulec/moto#5225]
Hi, This is question about validating the JWT signature.
I want to get jwks.json from [moto container](https://hub.docker.com/r/motoserver/moto).
I already created user pool, so I have the user pool id.

Does moto Docker image have an endpoint like `https://cognito-idp.{$region}.amazonaws.com/{$pool_id}/.well-known/jwks.json`?

**Comment by bblommers** [comment]
Hi @nakata72! Yes, you can call the same URL against the MotoServer: `http://localhost:5000/userpoolid/.well-known/jwks.json`

Note that, because we just lost context on which service and which region we want to talk to, this call requires an Authentication-header. Moto will use the value of this header to determine which service and region should handle this request.

An example:
`Authorization`: `AWS4-HMAC-SHA256 Credential=mock_access_key/20220524/us-east-1/cognito-idp/aws4_request, SignedHeaders=content-length;content-type;host;x-amz-date, Signature=asdf`

**Comment by nakata72** [comment]
Thanks @bblommers.
We tried your solution and got the results we wanted!

But I have a few questions.
Do I need to change the region like `us-east-1` included in the Authentication-header?
I tried two patterns that `us-east-1` and `ap-northeast-1`, and got same results.
In case of region us-east-1, I expected that got no result, because my created UserPool's region is at ap-northeast-1.
Is this correct behavor?

![image](https://user-images.githubusercontent.com/97861066/173720123-ca8e1473-635f-47f4-b1df-8038b372f57a.png)

**Comment by nakata72** [comment]
Thanks @bblommers.
We tried your solution and got the results we wanted!

But I have a few questions.
Do I need to change the region like us-east-1 included in the Authentication-header?
I tried two patterns that us-east-1 and ap-northeast-1, and got same results.
In case of region us-east-1, I expected that got no result, because my created UserPool's region is at ap-northeast-1.
Is this correct behavor?

![image](https://user-images.githubusercontent.com/97861066/173720491-cecd9b2b-28ca-477c-84c2-769391ed53f4.png)
